src/app/data-planes/components/DataPlaneSummary.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <div class="stack">
+    <DefinitionCard>
+      <template #title>
+        {{ t('http.api.property.status') }}
+      </template>
+
+      <template #body>
+        <div class="status-with-reason">
+          <StatusBadge :status="statusWithReason.status" />
+
+          <KTooltip
+            v-if="statusWithReason.reason.length > 0"
+            :label="statusWithReason.reason.join(', ')"
+            class="reason-tooltip"
+          >
+            <InfoIcon
+              :size="KUI_ICON_SIZE_30"
+              hide-title
+            />
+          </KTooltip>
+        </div>
+      </template>
+    </DefinitionCard>
+
+    <DefinitionCard>
+      <template #title>
+        {{ t('http.api.property.tags') }}
+      </template>
+
+      <template #body>
+        <TagList
+          v-if="dataPlaneTags.length > 0"
+          :tags="dataPlaneTags"
+        />
+
+        <template v-else>
+          {{ t('common.detail.none') }}
+        </template>
+      </template>
+    </DefinitionCard>
+
+    <DefinitionCard>
+      <template #title>
+        {{ t('http.api.property.dependencies') }}
+      </template>
+
+      <template #body>
+        <TagList
+          v-if="dataPlaneVersions !== null"
+          :tags="dataPlaneVersions"
+        />
+
+        <template v-else>
+          {{ t('common.detail.none') }}
+        </template>
+      </template>
+    </DefinitionCard>
+
+    <DefinitionCard>
+      <template #title>
+        {{ t('data-planes.routes.item.last_updated') }}
+      </template>
+
+      <template #body>
+        <template v-if="lastUpdatedTime">
+          {{ lastUpdatedTime }}
+        </template>
+
+        <template v-else>
+          {{ t('common.detail.none') }}
+        </template>
+      </template>
+    </DefinitionCard>
+
+    <DefinitionCard>
+      <template #title>
+        {{ t('data-planes.routes.item.certificate_info') }}
+      </template>
+
+      <template #body>
+        <template v-if="props.dataplaneOverview.dataplaneInsight?.mTLS?.certificateExpirationTime">
+          {{ formatIsoDate(props.dataplaneOverview.dataplaneInsight?.mTLS.certificateExpirationTime) }}
+        </template>
+
+        <template v-else>
+          {{ t('data-planes.routes.item.no_certificate') }}
+        </template>
```

Comment:
Just to note this seems to be almost exactly the same inline code as the code involved in the discussion here:

https://github.com/kumahq/kuma-gui/pull/1646/files#r1377489524

No action here, just a note, but I don't understand why its ok here, but not elsewhere.